frontend-css-concept/styles/styles.css

```
@@ -0,0 +1,346 @@
+    font-weight: 300;
+    font-size: 20px;
+}
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1, h2, h3, h4, h5, h6 {
+    font-family: Merriweather, sans-serif;
+}
+
+h1 {
+    font-size: 3rem;
+}
+
+h4 {
+    font-size: 1.5rem;
+}
+
+h5 {
+    font-size: 1rem;
+
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+.dev {
+    border: 2px solid red;
+}
+
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 100%;
+}
+
+.inner-container {
+    flex-basis: 1400px;
+    flex-shrink: 1;
+    flex-grow: 0;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    margin: 0 20px;
+}
+
+
+/* --------------- 3.2 Other --------------------------- */
+a,
+a:visited,
+a:link {
+    text-decoration: none;
+    color: var(--purple);
+}
+
+button {
+    font-weight: 300;
+    text-transform: uppercase;
+    color: white;
+    background-color: var(--lightred);
+    border-radius: 50px;
+    border: none;
+    box-shadow: 0 5px 5px 5px rgba(0, 0, 0, .16);
+    padding: 20px 40px;
+    margin: 20px 0;
+}
+
+button:hover {
+    cursor: pointer;
+}
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+header {
+    background-color: var(--blue);
+    color: white;
+    padding: 20px 0;
+    position: relative;
+}
+
+.hero-section__navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+}
+
+.hero-section__navbar ul {
+    flex-basis: 400px;
+    flex-shrink: 1;
+    flex-grow: 0;
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.hero-section__navbar a {
+    color: white;
+}
+
+.hero-section__content {
+    text-align: center;
+    font-size: 26px;
+    padding: 100px 0;
+    margin-bottom: 100px;
+}
+
+.header__squiggle {
+    background-image: url(./../assets/squiggle_line_stripe.svg);
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 80px;
+    height: 100px;
+    margin: 0 0 -20px 0;
+    align-self: flex-end;
+    position: absolute;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+#introduction-section {
+    background-color: white;
+    color: var(--purple);
+    text-align: center;
+    padding: 10px;
+}
+
+#introduction-section h4 {
+    margin: 10px;
+    padding: 6px;
+}
+.intro__link {
+    color: var(--black);
+    margin-top: 100px;
+    font-size: 16px;
+}
+/* --------------- 4.3 Projects ------------------- */
+
+#projects-section{
+    background-color: var(--lightgray);
+}
+.work-article {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    margin: 20px 20px;
```

Dit zou je nog korter kunnen schrijven naar
```suggestion
    margin: 20px;
```